<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBarTemp</class>
 <widget class="QWidget" name="frmBarTemp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="BarTemp" name="widget1">
     <property name="animation">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BarTemp" name="widget2">
     <property name="animation">
      <bool>true</bool>
     </property>
     <property name="animationStep">
      <double>1.000000000000000</double>
     </property>
     <property name="barColor">
      <color>
       <red>255</red>
       <green>107</green>
       <blue>107</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BarTemp" name="widget3">
     <property name="animation">
      <bool>true</bool>
     </property>
     <property name="animationStep">
      <double>2.000000000000000</double>
     </property>
     <property name="barColor">
      <color>
       <red>24</red>
       <green>189</green>
       <blue>155</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="BarTemp" name="widget4">
     <property name="animation">
      <bool>true</bool>
     </property>
     <property name="animationStep">
      <double>2.000000000000000</double>
     </property>
     <property name="barColor">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarTemp</class>
   <extends>QWidget</extends>
   <header>bartemp.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
